docs/pages/connect-your-client/tsh.mdx

```diff
@@ -715,13 +715,13 @@ $ tsh --proxy=work clusters
 ```
 
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem scope={["cloud"]} label="Teleport Enterprise">
 
-Assuming the Teleport Cloud tenant called `mytenant.teleport.sh` is configured with a few trusted
+Assuming the Teleport Enterprise tenant called `example.teleport.sh` is configured with a few trusted
```

Comment:
```suggestion
Assuming the Teleport Enterprise account called `example.teleport.sh` is configured with a few trusted
```
I think we're trying to move away from the "tenant" terminology.

Reply:
We may want to go through outside of this effort. It's used in lot of places. 

```
% grep -inr 'tenant' pages/*| wc -l
     109
```

For now I've replaced `mytenant.teleport.sh` with `example.teleport.sh`